js/base-jsload.js

```diff
@@ -59,10 +59,6 @@ function baseJsLoad () {
 	d20plus.js.pAddApiScripts = async () => {
 		d20plus.ut.log("Add Builtin API Scripts");
 
-		EXT_LIB_API_SCRIPTS.forEach(script => {
```

Comment:
This looks _very_ suspect. Is `EXT_LIB_API_SCRIPTS` not used? If so, can it also be deleted [assuming the code still exists elsewhere]? If not, why are you deleting it here?

(If this is just cleanup of unused junk, then, it's worth commenting that on the PR, fwiw, to head off these sorts of comments!)

Reply:
Somehow, this is exactly what's needed for the script to run. If this is not removed, or EXT_LIB_API_SCRIPTS is removed, the script breaks.

Reply:
I'm not sure I follow--the scripts that are tracked here in `EXT_LIB_API_SCRIPTS` should be:
```
VecMath.js
matrixMath.js
PathMath.js
```

If those aren't needed/used any more, then, fair enough--otherwise, they're loaded via the QPI "pipeline," and so, while it would be sensible to remove e.g. the shitty UI for the QPI, the pipeline itself needs to stay, no?

There are some obvious references to `PathMath` in the line-splitting tool code, at least, so I imagine it's still in use. I have a vague memory that the other two are both dependencies which `PathMath` requires, but that may be worth investigating

Reply:
Good point, didn't even notice that. I'll just undo some commits to make this change UI-only.